docs/src/pages/docs/guides/04-recruiter.mdx

```diff
@@ -1,18 +1,23 @@
 # Introduction
 
-In this guide, you’ll learn how Mastra helps you build workflows with LLMs. 
+In this guide, you’ll learn how Mastra helps you build workflows with LLMs.
 
-We’ll walk through creating a workflow that gathers information from a candidate’s resume, then branches to either a technical or behavioral question based on the candidate’s profile. Along the way, you’ll see how to structure workflow steps, handle branching, and integrate LLM calls.  
+We’ll walk through creating a workflow that gathers information from a candidate’s resume, then branches to either a technical or behavioral question based on the candidate’s profile. Along the way, you’ll see how to structure workflow steps, handle branching, and integrate LLM calls.
 
 Below is a concise version of the workflow. It starts by importing the necessary modules, sets up Mastra, defines steps to extract and classify candidate data, and then asks suitable follow-up questions. Each code block is followed by a short explanation of what it does and why it’s useful.
 
 ## 1. Imports and Setup
 
 You need to import Mastra tools and Zod to handle workflow definitions and data validation. This snippet also creates a Mastra instance for managing LLM interactions and orchestrating steps.
 
-```typescript
-import { Step, Workflow, Mastra } from '@mastra/core';
-import { z } from 'zod';
+```typescript filename="src/mastra/workflows/index.ts" copy
```

Comment:
Why are there two files? There shouldn't be two files. Everything should be in one file.

Reply:
That makes it simpler.

Reply:
Got it.